src/Chapter1.hs

```diff
@@ -436,6 +434,7 @@ Implement the function that takes an integer value and returns the next 'Int'.
   every type ｡.☆.*｡. No need to worry much about "error" here, just replace the
   function body with the proper implementation.
 -}
+next x =x +1
 next :: Int -> Int
 next x = error "next: not implemented!"
```

Comment:
You need to replace the code after `next x =` on the second line to solve this task. In Haskell, you are not allowed to have multiple definitions of the same function. Though in this particular case something different is happening, so this code compiles but behaves differently.

To solve this task successfully, you need to write the following function definition:

```haskell
next :: Int -> Int
next x = x + 1
```